### PR TITLE
[doc] Add instructions for the create-example-nft CLI Command

### LINKED ISSUE #1528: [P2] [documentation][Wallet CLI] Add documentation for creating example NFT
Add documentation for https://github.com/MystenLabs/sui/pull/1412

### PR DESCRIPTION
**Pull request by 666lcz**
We added a new command in https://github.com/MystenLabs/sui/pull/1412.

Closes #1528

### CODE REVIEW

File: doc/src/build/wallet.md [hunk 1]
```diff
@@ -195,25 +195,21 @@ Now start a new terminal since you have Sui running in the first terminal.
 
 The following commands are supported by the wallet:
 
-    active-address    Use this default address for commands when none is specified
-    addresses         Obtain the Addresses managed by the wallet
-    call              Call Move function
-    clear             Clear screen
-    echo              Write arguments to the console output
-    env               Print environment
-    exit              Exit the interactive shell
```
Comment (666lcz) [inline doc/src/build/wallet.md:204]: Don't think these exist

```
➜ wallet help
wallet
A Byzantine fault tolerant chain with low-latency finality and high throughput

USAGE:
    wallet [OPTIONS] [SUBCOMMAND]

OPTIONS:
        --config <CONFIG>    Sets the file storing the state of our user accounts (an empty one will
                             be created if missing)
    -h, --help               Print help information
    -i                       Start interactive wallet
        --json               Return command outputs in json format

SUBCOMMANDS:
    active-address        Default address used for commands when none specified
    addresses             Obtain the Addresses managed by the wallet
    call                  Call Move function
    create-example-nft    Create an example NFT
    gas                   Obtain all gas objects owned by the address
    help                  Print this message or the help of the given subcommand(s)
    merge-coin            Merge two coin objects into one coin
    new-address           Generate new address and keypair
    object                Get obj info
    objects               Obtain all objects owned by the address
    publish               Publish Move modules
    split-coin            Split a coin object into multiple coins
    switch                Switch active address
    sync                  Synchronize client state with authorities
    transfer              Transfer an object

```

Comment (patrickkuo) [reply doc/src/build/wallet.md:204]: These are interactive shell commands, it  exists when you print help inside the shell

Comment (Clay-Mysten) [reply doc/src/build/wallet.md:204]: Chris and I resolved this.

**Review by Clay-Mysten — APPROVED** [review]
Great addition, Chris! Sorry for the delay.

**Pull request merged**